examples/integration_with_error.py

```diff
@@ -39,115 +43,113 @@
 data.add_noise(fake_data_noise_std)
 data.reorder(["ph1", "ph2", "vd"])
 # at this point, the fake data has been generated
-data.ft(["ph1", "ph2"])
-fl.next("what does a usual error bar look like?")
-just_noise = nddata(r_[0:1:50j], "t")
-just_noise.data *= 0
-just_noise.add_noise(fake_data_noise_std)
-just_noise.set_error(fake_data_noise_std)
-fl.plot(just_noise, ".", capsize=6)
-# {{{ usually, we don't use a unitary FT -- this makes it unitary
-data /= 0.5 * 0.25  # the dt in the integral for both dims
-data /= sqrt(ndshape(data)["ph1"] * ndshape(data)["ph2"])  # normalization
-# }}}
 data.ft("t2", shift=True)
+data.ft(["ph1", "ph2"], unitary=True)
 dt = data.get_ft_prop("t2", "dt")
 # {{{ vector-normalize the FT
 data /= sqrt(ndshape(data)["t2"]) * dt
```

Comment:
without this line code fails

Reply:
I'm going to assume this comment is made obsolete by my recent push